src/utils/token.js

```diff
@@ -0,0 +1,3 @@
+export const getToken = () => localStorage.getItem('auth-token');
```

Comment:
I think you should use the `utils/localdata` utility here. You can probably just delete this file and use that utility directly inside of your components. If you really wanted to abstract something, you might just create a `utils/constants` and add a constant `const TOKEN_KEY = 'auth-token'` for use with storage.